docs/DeveloperGuide.md

```diff
@@ -287,14 +287,43 @@ _{Explain here how the data archiving feature will be implemented}_
 
-| Priority | As a …​                                    | I want to …​                 | So that I can…​                                                        |
-|----------|--------------------------------------------|------------------------------|------------------------------------------------------------------------|
-| `* * *`  | new user                                   | see usage instructions       | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person             |                                                                        |
-| `* * *`  | user                                       | delete a person              | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name        | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name         | locate a person easily                                                 |
+| Priority | As a …​                                   | I want to …​                                                               | So that I can…​                                                                                        |    
+|----------|-------------------------------------------|----------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------|
+| `* * *`  | social worker                             | delete a contact                                                           | remove the contact when I no longer serve them so that the contact list do not get too long            |
+| `* * *`  | social worker/new user                    | add contact with phone number                                              | remember the person i serve                                                                            |
+| `* * *`  | social worker/new user                    | add address                                                                | easily look up without needed to look up database/files                                                |
+| `* * *`  | social worker                             | view my list of contacts                                                   | so that i can see the beneficiaries i serve                                                            |
+| `* *`    | Overwhelmed with many households          | Take note of how long it has been since I visited each house               | Ensure that no house gets forgotten in my scheduling                                                   |
```

Comment:
nit: can standardise all to start with lower case.... applies to other lines too.